scripts/test_constraints_curve.py

```diff
@@ -0,0 +1,75 @@
+from functools import partial
+import compas
+from compas.geometry import Vector, Point, Line
+from compas_fd.datastructures import CableMesh
+from compas_fd.fd import fd_numpy
+
+from compas.geometry import Polyline, Bezier
+from compas.geometry import NurbsCurve
+
+from compas_fd.constraints import Constraint
+
+from compas_view2.app import App
+from compas_view2.objects import Object, MeshObject
+
+Object.register(CableMesh, MeshObject)
+
+mesh = CableMesh.from_obj(compas.get('faces.obj'))
+
+mesh.vertices_attribute('is_anchor', True, keys=list(mesh.vertices_where({'vertex_degree': 2})))
+mesh.vertices_attribute('t', 0.0)
+
+vertex = list(mesh.vertices_where({'x': 10, 'y': 10}))[0]
+points = [Point(10, 0, 0), Point(6, 13, -4), Point(-3, 16, 3), Point(0, 20, 0)]
+bezier = Bezier(points)
+points = bezier.locus(10)
+curve = NurbsCurve.from_interpolation(points)
```

Comment:
why not directly `NurbsCurve.from_points`?

Reply:
there is no reason really, definitely not needed. I just took it from the compas_occ examples file...